libsovtoken/Cargo.toml

```diff
@@ -28,7 +28,7 @@ log = "0.4.1"
 log-panics = "2.0.0"
 openssl = "0.9.24"
 rand = "0.4.2"
-rust-libindy-wrapper = "0.2.13"
+indy = "1.6.6-dev-1"
```

Comment:
only stable versions will be accepted as dependencies. Is this possible to do before the merge?

Reply:
It will be possible after Indy SDK release.

Reply:
We can change the version during our CI process

Reply:
I will need approval from @brentzundel and @burdettadam to make this merge since it's an exception to normal token merge standards.

Reply:
We can merge this PR right after new libindy stable.